apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -658,7 +663,8 @@ finish_request(Req, #base_state{reqid = ReqId,
         end
     catch
         X:Y ->
-            lager:error({X, Y, erlang:get_stacktrace()}),
+            lager:error("Error: ~p:~p. Stack trace follows.", [X, Y]),
+            lager:error("Stack Trace: ~p",  [erlang:get_stacktrace()]),
```

Comment:
Looks like it, I traced it back to chef_wm_base pre-merge[1], where it likely hadn't been touched for a while. However, it's possible that the different lager versions we went through handled it differently - our current version crashes noisily, but I think I remember the case of the disappearing messages as well. 

I feel like there's a way to do well-formatted stack traces, but maybe that was also part of `error_logger`. 

1) https://github.com/chef/oc_erchef/commit/5a623875635a9d3806504791008b07de3787ec83#diff-a9ec912a1c04bf1c21479c41ae09686dL214